CMakeLists.txt

```diff
@@ -75,9 +75,14 @@ if(WIN32)
     set(FreeOrion_INSTALL_LIBDIR ".")
 endif()
 
-if(APPLE)
+# If the current platform requires the FreeOrionSDK
+if(APPLE OR WIN32)
     set(FreeOrionSDK_VERSION 10)
-    set(FreeOrionSDK_FILE "FreeOrionSDK_${FreeOrionSDK_VERSION}_Clang-MacOSX-10.9-x86_64.tar.bz2")
+    if(WIN32)
+        set(FreeOrionSDK_FILE "FreeOrionSDK_${FreeOrionSDK_VERSION}_MSVC-v141-xp.zip")
+    else()
+        set(FreeOrionSDK_FILE "FreeOrionSDK_${FreeOrionSDK_VERSION}_Clang-MacOSX-10.9-x86_64.tar.bz2")
+    endif()
     if(NOT EXISTS "${CMAKE_BINARY_DIR}/${FreeOrionSDK_FILE}")
     file(DOWNLOAD
```

Comment:
I would like to have a possibility to use local SDK for testing purposes.

Reply:
Then place a local SDK at the correct location.